### PR TITLE
Send the video.js language to the tech

### PR DESCRIPTION
**Pull request by eXon**
I want to set the YouTube player (or any other iframe tech) in the same language than video.js. To do so, it needs to know the language option.

### CODE REVIEW

**Comment by heff** [comment]
lgtm.

Just interested, what does it look like to set the language of the underlying player?

File: src/js/player.js [hunk 1]
```diff
@@ -502,7 +502,8 @@ class Player extends Component {
       'autoplay': this.options_.autoplay,
       'preload': this.options_.preload,
       'loop': this.options_.loop,
-      'muted': this.options_.muted
+      'muted': this.options_.muted,
+      'language': this.options_.language
```
Comment (heff) [inline src/js/player.js]: You might want to use the player's language() getter here. It would probably provide some more consistency.

Comment (eXon) [inline src/js/player.js]: `language()` returns the player itself instead of the language. Is it normal?

Comment (heff) [inline src/js/player.js]: It should only return the player if an argument is passed to the function.
https://github.com/videojs/video.js/blob/dfd481037e19c2af546ccbc706b7fb42db1c7125/src/js/player.js#L2407

**Comment by eXon** [comment]
According to YouTube:

"The interface language is used for tooltips in the player and also affects the default caption track. Note that YouTube might select a different caption track language for a particular user based on the user's individual language preferences and the availability of caption tracks."

**Comment by heff** [comment]
Merged in, thanks!

**Pull request closed without merging**